tests/test_timezone.py

```diff
@@ -0,0 +1,208 @@
+import datetime
+from io import BytesIO
+
+import fastavro
+from fastavro.const import MCS_PER_SECOND, MLS_PER_SECOND
+from fastavro import _timezone as tz
+from fastavro._write_py import prepare_timestamp_micros
+from fastavro._write_py import prepare_timestamp_millis
+import pytest
+
+
+schema = {
+    "fields": [
+          {
+              "name": "timestamp-millis",
+              "type": {'type': 'long', 'logicalType': 'timestamp-millis'}
+          },
+          {
+              "name": "timestamp-micros",
+              "type": {'type': 'long', 'logicalType': 'timestamp-micros'}
+          },
+    ],
+    "namespace": "namespace",
+    "name": "name",
+    "type": "record"
+}
+
+
+# Test Time Zone with fixed offset and 1 hr DST
```

Comment:
I think either this comment or the one for the `TDTTzinfo` is incorrect? They are both the same but the classes are different.

Reply:
++. 

Side note: from [Python 3 API](https://docs.python.org/3/library/datetime.html#datetime.timezone.dst) it looks like DST was something that kinda got dropped in Python 3 (i.e. always return None). By contrast, [Python 2 API](https://docs.python.org/2/library/datetime.html#datetime.tzinfo.dst) it seems that when timezones were just an Abstract Base Class, there was some notion of useful information from DST. 

Given this, I think it's worth removing any test for daylight savings time compatibility given that this seems to be kinda unsupported in Python3. I'm open to feedback here though.

Reply:
That sounds fine to me. I'll defer to @kkirsanov and your judgements as I don't really use the timezone stuff.